docs/status.md

```diff
@@ -0,0 +1,95 @@
+---
+title: Components and Applications
+layout: default
+nav_order: 4
+---
+
+# Myna Components and Applications
+{: .no_toc }
+
+## Table of Contents
+{: .no_toc .text-delta }
+
+1. TOC
+{:toc}
+
+Myna version: 1.1.0.dev0
+
+This page contains the list of available classes within Myna for workflow components, input and output data, and applications.
```

Comment:
The "status.md" file is generated programmatically by running `myna status --ghpages --output docs/status.md`. So maybe that can be added as a CI?

This file is a bit cleaner than just a straight API lookup, since it filters out a lot of less essential info.